chore(Android): Add troubleshooting entry about Sentry.init (#11147)
getsentry/sentry-docs: docs/platforms/android/troubleshooting/index.mdx

sidebar_order: 9000
---

## Obfuscated stack traces, missing Android-specific context

> If stack traces don't get unobfuscated properly, even though <PlatformLink to="/enhance-errors/proguard/">mapping files are uploaded correctly</PlatformLink>, or if Android context information is missing from events, a possible reason is that the SDK is initialized with `Sentry.init` instead of `SentryAndroid.init`.

Reviewer: ```suggestion
If stack traces don't get deobfuscated properly, even though <PlatformLink to="/enhance-errors/proguard/">mapping files are uploaded correctly</PlatformLink>, or if Android context information is missing from events, a possible reason is that the SDK is initialized with `Sentry.init` instead of `SentryAndroid.init`.
```

<Alert level="warning">
For manual initialization on Android, <PlatformLink to="/configuration/manual-init/#manual-initialization">always use `SentryAndroid.init`</PlatformLink>.
</Alert>

> From SDK version `8.0.0` on, using `Sentry.init` on Android will cause a compile-time exception.

Reviewer: ```suggestion
From SDK version `8.0.0` on, using `Sentry.init` on Android will throw an `IllegalArgumentException`.
```
